### ISSUE TITLE
Map out tempo output files to NeoantigenEditing input

### ISSUE THREAD

**Issue by nikhil** [mskcc/NeoAntigen-Pipeline#4]
Of the 4 indicated input files:

#### iedb.fasta 
a resource

#### epitope_distance_model_parameters.json 
a resource created by the lab
https://github.com/LukszaLab/NeoantigenEditing/blob/0126b82b1b4c0d44b6e3f5559c43e6022b8cdbd3/data/epitope_distance_model_parameters.json

#### fitness_weights.txt
no idea what this is
https://github.com/LukszaLab/NeoantigenEditing/blob/0126b82b1b4c0d44b6e3f5559c43e6022b8cdbd3/data/fitness_weights.txt

#### patient_data 
Mutation data under a clonal tree
https://github.com/LukszaLab/NeoantigenEditing/blob/0126b82b1b4c0d44b6e3f5559c43e6022b8cdbd3/data/Patient_data/11-LTS/Primary/11_LTS_primary_tumor.json

After reading the methods of the paper it looks like you need:
HLA typing, they use OptiType
Clones created by PhyloWGS
NetMHC?

Basically, we first need to know what information to gather to form the input files, and then we can focus on formatting it properly

**Comment by nikhil** [comment]
looks like we will run only 1 and 2, might need to fork the code to get the specific calculations we need

**Comment by nikhil** [comment]
Example of a fork someone else did: https://github.com/LukszaLab/NeoantigenEditing/compare/main...LCR-BCCRC-Steidl-Lab:NeoantigenEditing_fork:main#diff-7849ed4b5e38dcf9ebb66e5159c59181e99a3de20f7ff1f619881c723532c5bcR329

**Comment by nikhil** [comment]
It looks like the input of this file: https://github.com/LukszaLab/NeoantigenEditing/blob/main/data/Patient_data/11-LTS/Primary/11_LTS_primary_tumor.json, is just the phylowgs + OptiType

**Comment by johnoooh** [comment]
I'll work on this as well

**Comment by nikhil** [comment]
https://docs.google.com/document/d/1gj4UgrihULmM5DyrzT8yAJty3W5gGHAuxfiQeoq8YPo/edit?usp=sharing
